File: app/src/main/java/com/infobox/hasnat/ume/ume/Model/Requests.java
package com.infobox.hasnat.ume.ume.Model;

public class Requests {

    private String user_name, user_status, user_thumb_image;

    public Requests() {
    }

    public Requests(String user_name, String user_status, String user_thumb_image) {
        this.user_name = user_name;
        this.user_status = user_status;
        this.user_thumb_image = user_thumb_image;
    }

    public String getUser_name() {
        return user_name;
    }

    public void setUser_name(String user_name) {
        this.user_name = user_name;
    }

    public String getUser_status() {
        return user_status;
    }

    public void setUser_status(String user_status) {
        this.user_status = user_status;
    }

    public String getUser_thumb_image() {
        return user_thumb_image;
    }

    public void setUser_thumb_image(String user_thumb_image) {
        this.user_thumb_image = user_thumb_image;
    }
}

File: app/src/main/java/com/infobox/hasnat/ume/ume/Model/Requests.kt
package com.infobox.hasnat.ume.ume.Model

class Requests {

    var user_name: String? = null
    var user_status: String? = null
    var user_thumb_image: String? = null

    constructor() {}

    constructor(user_name: String, user_status: String, user_thumb_image: String) {
        this.user_name = user_name
        this.user_status = user_status
        this.user_thumb_image = user_thumb_image
    }
}
